### DISCUSSION TITLE
"Throw focused window to space left/right" doesn't work for me anymore

### DISCUSSION THREAD

**Discussion by projeffboy** [ianyh/Amethyst#1659] [General]
I have three monitors, from left to right: macbook, dell monitor, ipad with sidecar. For each monitor, I have multiple virtual desktops/spaces. When I want to switch between monitors, I use "Throw focused window to screen X", but when I want to switch between virtual desktops/spaces, I use "Throw focused window to space left/right". The former still works, but the latter doesn't work for me anymore. Often times, it would switch to another space but immediately go back to its original space.

I am on Sonoma 14.5

**Comment by larry-lime** [comment]
I have the same issue. Throwing focused window between spaces is broken for me too.

**Reply by DanCates** [reply]
@projeffboy and @larry-lime 
This is fixed in the 0.21.1 update that was released the other week. I had the same issue as both of you after upgrading MacOS to 14.5. I couldn't throw windows across spaces but now I can with this 0.21.1 Amethyst release. 

Specifically, I just tested the monitor throwing scenario you spoke of in your original post, @projeffboy . [Here's my video of it.](https://www.loom.com/share/3959f5bc700449c4b9eb1085b5c4f22f?sid=e33eccdf-f78d-4b9e-a32f-fa420dae339d) At timestamp 0:20 you can specifically see me throwing between spaces within two of my monitors.

**Comment by DamhertPro** [comment]
I still have the same problem under macOS Ventura 13.6.7 in version 0.21.1 did I missed something?

**Comment by ianyh** [comment]
See https://github.com/ianyh/Amethyst/discussions/1682
